src/ci/azure-pipelines/auto.yml

```diff
@@ -53,6 +53,7 @@ jobs:
       dist-powerpc-linux: {}
       dist-powerpc64-linux: {}
       dist-powerpc64le-linux: {}
+      dist-riscv64-linux: {}
```

Comment:
You also need to add the builder to GHA.

Reply:
Added.